portal/code/cx-portal/src/components/shared/frame/Footer/index.tsx

```diff
@@ -1,20 +1,49 @@
-import { Navigation } from 'cx-portal-shared-components'
+import { useEffect, useState } from "react";
+import { Navigation, IconButton } from 'cx-portal-shared-components'
+import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward';
 import { useTranslation } from 'react-i18next'
 import { NavLink } from 'react-router-dom'
 import './Footer.scss'
 
 export const Footer = ({ pages }: { pages: string[] }) => {
   const { t } = useTranslation()
-
   const items = pages.map((page) => ({
     to: page,
     title: t(`pages.${page}`),
   }))
+  const scrollToTop = () => {
+    window.scrollTo({
+      top: 0,
+      behavior: "smooth"
+    });
+  };
+
+  const [showScrollToTop, setShowScrollToTop] = useState(false);
+
+  const toggleVisibility = () => {
+    if (window.pageYOffset > 350) {
```

Comment:
I suggest to replace this if-else with: `setShowScrollToTop(window.pageYOffset > 350)`

Reply:
Find ich super! Danke :)